src/pybind/mgr/dashboard/openapi.yaml

```diff
@@ -3116,6 +3116,55 @@ paths:
       - jwt: []
       tags:
       - Cephfs
+  /api/cephfs/{fs_id}/rename-path:
```

Comment:
so did the script generate `rename-path`? i hope this is correct since the api name is `rename_path` right?

Reply:
endpoint use `'PUT', path='/rename-path'`, for example:
```
curl -X PUT "0.0.0.0:8080/api/cephfs/1/rename-path" -H "Accept: application/vnd.ceph.api.v1.0+json" -H "Content-Type: application/json" -d '{"src_path": "/dir9", "dst_path": "/dir10"}' -H  "Authorization: Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJleHAiOjE3MjAxMDI0OTYsImlhdCI6MTcyMDA3MzY5NiwiaXNzIjoiY2VwaC1kYXNoYm9hcmQiLCJqdGkiOiJiOTQxOTI1My0zYmVmLTQ4NmEtOGQ5ZS1mNDdhZDM2NDI2Y2QiLCJ1c2VybmFtZSI6InlpdGVndSJ9.g9uG-Cd5xPX0v7IXvS6uJg4MNvQKnOGFAHFhIvUBTRk"
```

Reply:
Yeah, its not the function name but the endpoint name. Usually they are the same but it could be different too when the path is explicitly provided. So this should be correct.

Reply:
alright